generative/maisi/scripts/diff_model_create_training_data.py

```diff
@@ -0,0 +1,204 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import json
+import os
+from pathlib import Path
+
+import nibabel as nib
+import numpy as np
+import torch
+import torch.distributed as dist
+import torch.nn.functional as F
+
+import monai
+from monai.transforms import Compose
+from monai.utils import set_determinism
+
+from utils import define_instance, load_autoencoder_ckpt
+
+# Set the random seed for reproducibility
+torch.manual_seed(0)
+np.random.seed(0)
+set_determinism(seed=0)
```

Comment:
```suggestion
set_determinism(seed=0)
```

`set_determinism` already include the above two lines, we can remove them.

Reply:
Hi @KumoLiu, I have updated the script. Please proceed. Thanks.